slick.grid.js

```diff
@@ -1973,133 +2027,119 @@ if (typeof Slick === "undefined") {
           if (options.frozenBottom) {
-            $viewportScrollContainerX = $viewportBottomR;
-            $viewportScrollContainerY = $viewportTopR;
+            _viewportScrollContainerX = _viewportBottomR;
+            _viewportScrollContainerY = _viewportTopR;
           } else {
-            $viewportScrollContainerX = $viewportScrollContainerY = $viewportBottomR;
+            _viewportScrollContainerX = _viewportScrollContainerY = _viewportBottomR;
           }
         } else {
-          $viewportScrollContainerX = $viewportScrollContainerY = $viewportTopR;
+          _viewportScrollContainerX = _viewportScrollContainerY = _viewportTopR;
         }
       } else {
-        $headerScrollContainer = $headerScrollerL;
-        $headerRowScrollContainer = $headerRowScrollerL;
-        $footerRowScrollContainer = $footerRowScrollerL;
+        _headerScrollContainer = _headerScrollerL;
+        _headerRowScrollContainer = _headerRowScrollerL;
+        _footerRowScrollContainer = _footerRowScrollerL;
 
         if (hasFrozenRows) {
           if (options.frozenBottom) {
-            $viewportScrollContainerX = $viewportBottomL;
-            $viewportScrollContainerY = $viewportTopL;
+            _viewportScrollContainerX = _viewportBottomL;
+            _viewportScrollContainerY = _viewportTopL;
           } else {
-            $viewportScrollContainerX = $viewportScrollContainerY = $viewportBottomL;
+            _viewportScrollContainerX = _viewportScrollContainerY = _viewportBottomL;
           }
         } else {
-          $viewportScrollContainerX = $viewportScrollContainerY = $viewportTopL;
+          _viewportScrollContainerX = _viewportScrollContainerY = _viewportTopL;
         }
       }
     }
 
     function measureCellPaddingAndBorder() {
-      var el;
-      var h = ["borderLeftWidth", "borderRightWidth", "paddingLeft", "paddingRight"];
-      var v = ["borderTopWidth", "borderBottomWidth", "paddingTop", "paddingBottom"];
-
-      // jquery prior to version 1.8 handles .width setter/getter as a direct css write/read
-      // jquery 1.8 changed .width to read the true inner element width if box-sizing is set to border-box, and introduced a setter for .outerWidth
-      // so for equivalent functionality, prior to 1.8 use .width, and after use .outerWidth
-      var verArray = $.fn.jquery.split('.');
-      jQueryNewWidthBehaviour = (verArray[0]==1 && verArray[1]>=8) ||  verArray[0] >=2;
+      const h = ["borderLeftWidth", "borderRightWidth", "paddingLeft", "paddingRight"];
+      const v = ["borderTopWidth", "borderBottomWidth", "paddingTop", "paddingBottom"];
+      const header = _headers[0];
 
-      el = $("<div class='ui-state-default slick-header-column' style='visibility:hidden'>-</div>").appendTo($headers);
       headerColumnWidthDiff = headerColumnHeightDiff = 0;
-      if (el.css("box-sizing") != "border-box" && el.css("-moz-box-sizing") != "border-box" && el.css("-webkit-box-sizing") != "border-box") {
```

Comment:
the `if` condition is missing and is causing some styling issues on my side, I believe it should be
```ts
if (style["box-sizing"] != "border-box" && style["-moz-box-sizing"] != "border-box" && style["-webkit-box-sizing"] != "border-box") {
  //...
```